.github/workflows/rails-ci.yml

```
@@ -27,31 +30,69 @@ jobs:
+          DYNAMIC_CONFIG_FILE_PATH: config/dynamicconfig/development.yaml
+          TEMPORAL_CLI_ADDRESS: localhost:7233
+        ports:
+          - 7233:7233
+        options: >-
+          --health-cmd "tctl --address localhost:7233 namespace list"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
 
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)